modules/porous_flow/test/tests/capillary_pressure/brooks_corey1.i

```diff
@@ -143,7 +143,7 @@
 [Executioner]
   type = Steady
   solve_type = NEWTON
-  nl_abs_tol = 1e-12
+  nl_abs_tol = 1e-6
```

Comment:
I presume that nl_abs_tol=1E-6 is the lowest you can go.  It's a shame: my computer gives |R|=1E-15.